modules/client-configuration/pages/clients-microos.adoc

```diff
@@ -0,0 +1,144 @@
+[[clients-microos]]
+= Registering {opensuse}{nbsp}{microos} Clients
```

Comment:
Consider the macro change I suggested above, same for the rest of `{opensuse}{nbsp}{microos}` references.